docs/AuthenticationMethods.md

```diff
@@ -43,22 +48,23 @@ The connector will look for the following secret names:
  * **kustoAppAuthority**
   AAD authentication authority. This is the AAD Directory (tenant) ID.
 
-###Read
+### Read Parameters
```

Comment:
### Read Storage Parameters
-> Note: these parameters are only required when working in "scale" reading mode. For details, refer to [](add a link to the relevant section here)